Not sure if test failures are related to this change:
```
FAIL: custom.icds_reports.tests.test_aggregation_script:AggLsAggregationTest.test_agg_ls_2017_05_01_1
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/mnt/commcare-hq-ro/custom/icds_reports/tests/test_aggregation_script.py", line 365, in test_agg_ls_2017_05_01_1
    sort_key=['state_id', 'district_id', 'block_id', 'supervisor_id']
  File "/mnt/commcare-hq-ro/custom/icds_reports/tests/test_aggregation_script.py", line 138, in _load_and_compare_data
    self._load_csv(path)
  File "/mnt/commcare-hq-ro/custom/icds_reports/tests/test_aggregation_script.py", line 94, in _fasterAssertListEqual
    self.fail('Lists are not equal')
AssertionError: Lists are not equal

======================================================================
FAIL: corehq.messaging.smsbackends.tropo.tests.test_log_call:TropoLogCallTestCase.test_log_call
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/mnt/commcare-hq-ro/corehq/util/test_utils.py", line 262, in inner
    return helper(*args, **kwargs)
  File "/mnt/commcare-hq-ro/corehq/util/test_utils.py", line 234, in __call__
    call_with_settings(fn_with_pre_and_post, run_config.settings, args, kwargs)
  File "/mnt/commcare-hq-ro/corehq/util/test_utils.py", line 248, in call_with_settings
    fn(*args, **kwargs)
  File "/mnt/commcare-hq-ro/corehq/util/test_utils.py", line 230, in fn_with_pre_and_post
    self.fn(*args, **kwargs)
  File "/mnt/commcare-hq-ro/corehq/apps/ivr/tests/util.py", line 72, in test_log_call
    self.assertEqual(call.couch_recipient, case.case_id)
AssertionError: u'2755cbddcce8469496cb3c2025980b21' != u'40a8faa3730c40d1a272050dc324261b'
- 2755cbddcce8469496cb3c2025980b21
+ 40a8faa3730c40d1a272050dc324261b
-------------------- >> begin captured stdout << ---------------------
<function test_log_call at 0x7f4a6240a500> failed with the following settings:
settings.TESTS_SHOULD_USE_SQL_BACKEND = True
--------------------- >> end captured stdout << ----------------------

--- a/corehq/apps/app_manager/models.py
+++ b/corehq/apps/app_manager/models.py
@@ -1212,7 +1212,8 @@ class FormBase(DocumentSchema):
         return xform.render()
 
     @time_method()
-    @quickcache(['self.source', 'langs', 'include_triggers', 'include_groups', 'include_translations'])
+    @quickcache(['self.source', 'langs', 'include_triggers', 'include_groups', 'include_translations'],
+                timeout=24 * 60 * 60)
     def get_questions(self, langs, include_triggers=False,
                       include_groups=False, include_translations=False):
         try:

